### DISCUSSION TITLE
How can I inject container registry credentials and kubernetes api credentials into the operator agent container directly

### DISCUSSION THREAD

**Discussion by karchf** [getporter/porter#3093] [Q&A]
Bit of a strange situation. Due to security constraints, IronBank approved containers can be used, but the operator cannot. Need to utilize the IB approved agent to publish a bundle into a container registry from local disk and then install the bundle into a kubernetes cluster.

I know that the operator has to inject the credentials into the agent container via some means. I went to spin up the operator to reverse engineer the means of credential injection, but it seems the quickstart for the operator is currently broken.

In addition, utilizing podman instead of docker, so no docker daemon is available. Any advice?

**Comment by bdegeeter** [comment]
Is this what you're looking for? https://porter.sh/docs/operator/quickstart/#private-bundle-registries

You can add credentials to access a private registry.
